### ISSUE TITLE
[beta] Introduction to HTML 5 Elements - Correct code fails test case 3

### ISSUE THREAD

**Issue by P1xt** [freeCodeCamp/freeCodeCamp#12774]
![basic html and html5 introduction to html5 elements freecodecamp](https://cloud.githubusercontent.com/assets/322175/22208681/545e3c26-e149-11e6-83cf-f7fc45988503.png)

Challenge [introduction-to-html5-elements](http://beta.freecodecamp.com/en/challenges/basic-html-and-html5/introduction-to-html5-elements) has an issue.
User Agent is: <code>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/55.0.2883.87 Chrome/55.0.2883.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html


<h2>CatPhotoApp</h2>
<main>
<p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
</main>

```

This has been reported before but the issue was closed with a link to a completely unrelated issue days ago, there are no currently open bug reports specifically addressing this issue.

**Comment by P1xt** [comment]
Errors show in chrome desktop when this occurred:

![desctop screenshot](https://cloud.githubusercontent.com/assets/322175/22209282/8eb3ad00-e14b-11e6-9b11-33e6ddd63293.png)

**Comment by P1xt** [comment]
Refused to load the script 'data:application/javascript;base64,KGZ1bmN0aW9uKCkgewoJLy8gaHR0cHM6Ly9kZXZl…07Cgl9OwoJZ2EucmVtb3ZlID0gbm9vcGZuOwoJd2luZG93W2dhTmFtZV0gPSBnYTsKfSkoKTs=' because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'unsafe-inline' *.google-analytics.com *.gstatic.com https://*.cloudflare.com *.cloudflare.com https://*.gitter.im https://*.cdnjs.com *.cdnjs.com https://*.jsdelivr.com *.jsdelivr.com *.twimg.com https://*.twimg.com *.youtube.com *.ytimg.com https://*.optimizely.com 'self'".

bundle-0f6b2db….js:1 Uncaught Error: Minified React error #32; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=32&args[]=45 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.(…)n @ bundle-0f6b2db….js:1s @ bundle-0f6b2db….js:6c @ bundle-0f6b2db….js:6getHostNode @ bundle-0f6b2db….js:48getHostNode @ bundle-0f6b2db….js:8_updateChildren @ bundle-0f6b2db….js:48updateChildren @ bundle-0f6b2db….js:48_updateDOMChildren @ bundle-0f6b2db….js:48updateComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:8_updateRenderedComponent @ bundle-0f6b2db….js:47_performComponentUpdate @ bundle-0f6b2db….js:47updateComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:8updateChildren @ bundle-0f6b2db….js:47_reconcilerUpdateChildren @ bundle-0f6b2db….js:48_updateChildren @ bundle-0f6b2db….js:48updateChildren @ bundle-0f6b2db….js:48_updateDOMChildren @ bundle-0f6b2db….js:48updateComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:8_updateRenderedComponent @ bundle-0f6b2db….js:47_performComponentUpdate @ bundle-0f6b2db….js:47updateComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:8updateChildren @ bundle-0f6b2db….js:47_reconcilerUpdateChildren @ bundle-0f6b2db….js:48_updateChildren @ bundle-0f6b2db….js:48updateChildren @ bundle-0f6b2db….js:48_updateDOMChildren @ bundle-0f6b2db….js:48updateComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:8updateChildren @ bundle-0f6b2db….js:47_reconcilerUpdateChildren @ bundle-0f6b2db….js:48_updateChildren @ bundle-0f6b2db….js:48updateChildren @ bundle-0f6b2db….js:48_updateDOMChildren @ bundle-0f6b2db….js:48updateComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:8_updateRenderedComponent @ bundle-0f6b2db….js:47_performComponentUpdate @ bundle-0f6b2db….js:47updateComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:8_updateRenderedComponent @ bundle-0f6b2db….js:47_performComponentUpdate @ bundle-0f6b2db….js:47updateComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:8updateChildren @ bundle-0f6b2db….js:47_reconcilerUpdateChildren @ bundle-0f6b2db….js:48_updateChildren @ bundle-0f6b2db….js:48updateChildren @ bundle-0f6b2db….js:48_updateDOMChildren @ bundle-0f6b2db….js:48updateComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:8_updateRenderedComponent @ bundle-0f6b2db….js:47_performComponentUpdate @ bundle-0f6b2db….js:47updateComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:8updateChildren @ bundle-0f6b2db….js:47_reconcilerUpdateChildren @ bundle-0f6b2db….js:48_updateChildren @ bundle-0f6b2db….js:48updateChildren @ bundle-0f6b2db….js:48_updateDOMChildren @ bundle-0f6b2db….js:48updateComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:48receiveComponent @ bundle-0f6b2db….js:8_updateRenderedComponent @ bundle-0f6b2db….js:47_performComponentUpdate @ bundle-0f6b2db….js:47updateComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:47receiveComponent @ bundle-0f6b2db….js:8_updateRenderedComponent @ bundle-0f6b2db….js:47_performComponentUpdate @ bundle-0f6b2db….js:47updateComponent @ bundle-0f6b2db….js:47performUpdateIfNecessary @ bundle-0f6b2db….js:47performUpdateIfNecessary @ bundle-0f6b2db….js:8s @ bundle-0f6b2db….js:7perform @ bundle-0f6b2db….js:10perform @ bundle-0f6b2db….js:10perform @ bundle-0f6b2db….js:7A @ bundle-0f6b2db….js:7closeAll @ bundle-0f6b2db….js:10perform @ bundle-0f6b2db….js:10batchedUpdates @ bundle-0f6b2db….js:23i @ bundle-0f6b2db….js:7_renderNewRootComponent @ bundle-0f6b2db….js:24_renderSubtreeIntoContainer @ bundle-0f6b2db….js:24render @ bundle-0f6b2db….js:24(anonymous function) @ bundle-0f6b2db….js:53(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:6Ye.AnonymousObservable.r._subscribe @ bundle-0f6b2db….js:6Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2t.subscribeCore @ bundle-0f6b2db….js:4(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:2Ye.ObservableBase.r._subscribe @ bundle-0f6b2db….js:2Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2t.next @ bundle-0f6b2db….js:3Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1n.next @ bundle-0f6b2db….js:6Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2Ye.FlatMapObservable.n.next @ bundle-0f6b2db….js:2Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1n.next @ bundle-0f6b2db….js:6Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2n.next @ bundle-0f6b2db….js:3Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1n.next @ bundle-0f6b2db….js:6Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2n.next @ bundle-0f6b2db….js:3Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1n.next @ bundle-0f6b2db….js:6Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2r.next @ bundle-0f6b2db….js:4Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1n.next @ bundle-0f6b2db….js:6Ye.internals.AbstractObserver.t.onNext @ bundle-0f6b2db….js:2_subscribe @ bundle-0f6b2db….js:6Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:6Ye.AnonymousObservable.r._subscribe @ bundle-0f6b2db….js:6Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2t.subscribeCore @ bundle-0f6b2db….js:4(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:2Ye.ObservableBase.r._subscribe @ bundle-0f6b2db….js:2Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2t.subscribeCore @ bundle-0f6b2db….js:3(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:2Ye.ObservableBase.r._subscribe @ bundle-0f6b2db….js:2Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2t.subscribeCore @ bundle-0f6b2db….js:3(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:2Ye.ObservableBase.r._subscribe @ bundle-0f6b2db….js:2Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2Ye.FlatMapObservable.t.subscribeCore @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:2Ye.ObservableBase.r._subscribe @ bundle-0f6b2db….js:2Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2t.subscribeCore @ bundle-0f6b2db….js:3(anonymous function) @ bundle-0f6b2db….js:1n @ bundle-0f6b2db….js:2bn.invokeCore @ bundle-0f6b2db….js:2bn.invoke @ bundle-0f6b2db….js:2t @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1n.schedule @ bundle-0f6b2db….js:2Ye.ObservableBase.r._subscribe @ bundle-0f6b2db….js:2Ye.Observable.Kn.subscribe.Kn.forEach @ bundle-0f6b2db….js:2(anonymous function) @ bundle-0f6b2db….js:1t @ bundle-0f6b2db….js:1(anonymous function) @ bundle-0f6b2db….js:1(anonymous function) @ bundle-0f6b2db….js:1

**Comment by ananaymital** [comment]
@P1xt Thanks for the reporting this issue. 
Turns out this has already fixed! 🎉 The fix has been done(#12656), but has not been deployed to the beta yet. You'll see it appear the next time beta is updated.

Thanks again for reporting this issue, and happy coding!

**Comment by dhcodes** [comment]
@P1xt closing because I am 99% sure this will be fixed when the site is updated with this PR: https://github.com/freeCodeCamp/freeCodeCamp/pull/12663

Please reopen if it's not.

**Comment by P1xt** [comment]
Just out of curiosity, why would you close this before it can be QA'd on beta and confirmed fixed.

**Comment by dhcodes** [comment]
@P1xt you're welcome to reopen it; I mostly closed it so contributors don't invest time reading through it only to realize there's already a PR merged that will probably fix it.

**Comment by dhcodes** [comment]
@P1xt As a follow-up, I read your post on the forum and don't want you to feel discouraged in anyway from submitting new issues. I'm probably guilty of sometimes being overzealous in closing them. I went ahead and confirmed that this issue is fixed on staging. Below is a screenshot.
[!screen](http://i.imgur.com/lH3SY7C.png)

Cheers.

**Comment by P1xt** [comment]
My only real concern with the overzealous closing of issues when they've yet to be confirmed "fixed" is that last time there was a big push from beta to live everything broke and the issues had all been discovered in beta, then either ignored or closed.

From a process perspective ... noting that a particularly PR will likely fix an issue is an awesome idea. Closing the issue when it's not yet confirmed as fixed leaves giant holes in the process where things get reported and no one ever actually QAs the fix so the bug persists.

**Comment by dhcodes** [comment]
@P1xt I see your point.

**Comment by systimotic** [comment]
@P1xt It looks like this is a duplicate of #12656. Can you confirm?

**Comment by P1xt** [comment]
@systimotic There's a slight possibility but it's quite remote. The issue on #12656 is one I've seen on a number of challenges where the grader just stops accessing whatever it needs to populate the various sections of the page. This issue, on the other hand is challenge specific, wherein the platform seems to be working fine but this particular challenge is incorrectly mis-grading one of the test cases and not registering a correct answer.

**Comment by HKuz** [comment]
I think the first mention of the challenge-specific test issue was over in #12653 - which was closed automatically when PR #12663 was QA'd and merged. I think part of the confusion is that when subsequent issues about this specific challenge and the test were raised, I referenced the PR instead of marking them as duplicate issues 😞  . I'll go back and edit my comments on those, so it's easier for folks QA'ing beta to see if issues were already raised and to follow the chain of events.

Thanks for your patience, @P1xt - I'm not sure when the next update to include the recent changes in `staging` is to the beta site. Hopefully when that happens, we can confirm the fix works for the broader community.
